src/ipc/UdsOp.cc

```diff
@@ -163,8 +164,10 @@ void Ipc::UdsSender::DelayedRetry(void *data)
     if (UdsSender *us = dynamic_cast<UdsSender*>(ptr->valid())) {
         // get back inside AsyncJob protection by scheduling an async job call
         typedef NullaryMemFunT<Ipc::UdsSender> Dialer;
-        AsyncCall::Pointer call = JobCallback(54, 4, Dialer, us, Ipc::UdsSender::delayedRetry);
-        ScheduleCallHere(call);
+        CallBack(us->codeContext, [&us] {
+            AsyncCall::Pointer call = JobCallback(54, 4, Dialer, us, Ipc::UdsSender::delayedRetry);
+            ScheduleCallHere(call);
```

Comment:
If we can use CallJobHere(), like in the other cases in this PR, please do that.

Reply:
Done.